examples/blame_example.ipynb

```diff
@@ -1,310 +1,240 @@
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 12,
-   "metadata": {},
-   "outputs": [
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "L0ufxMO8NdvX"
+      },
+      "outputs": [],
+      "source": [
+        "collisions = []\n",
+        "collision_index = None\n",
+        "is_found_collision = False\n",
+        "\n",
+        "# Simulate with `initialize`, `drive` and `light` until there are collisions.\n",
+        "for _ in range(20): #Attempt 20 simulations looking for a collision\n",
+        "    light_response = iai.light(location=location)\n",
+        "\n",
+        "    response = iai.initialize(\n",
+        "        location=location,\n",
+        "        agent_count=15,\n",
+        "        get_birdview=True,\n",
+        "        traffic_light_state_history=[light_response.traffic_lights_states]\n",
+        "    )\n",
+        "    agent_attributes = response.agent_attributes\n",
+        "    scene_plotter.initialize_recording(response.agent_states,\n",
+        "                                       agent_attributes=agent_attributes,\n",
+        "                                       traffic_light_states=light_response.traffic_lights_states)\n",
+        "\n",
+        "    agent_state_history = []\n",
+        "    traffic_light_state_history = []\n",
+        "\n",
+        "    # 20-second scene\n",
+        "    for t in range(100):\n",
+        "        light_response = iai.light(location=location, recurrent_states=light_response.recurrent_states)\n",
+        "        response = iai.drive(\n",
+        "            location=location,\n",
+        "            agent_attributes=agent_attributes,\n",
+        "            agent_states=response.agent_states,\n",
+        "            recurrent_states=response.recurrent_states,\n",
+        "            get_birdview=False,\n",
+        "            traffic_lights_states=light_response.traffic_lights_states,\n",
+        "            get_infractions=True,\n",
+        "            random_seed=1\n",
+        "        )\n",
+        "        scene_plotter.record_step(response.agent_states, traffic_light_states=light_response.traffic_lights_states)\n",
+        "        agent_state_history.append(response.agent_states)\n",
+        "        traffic_light_state_history.append(light_response.traffic_lights_states)\n",
+        "\n",
+        "        for agent, infractions in enumerate(response.infractions):\n",
+        "            if infractions.collisions: collisions.append(agent)\n",
+        "            if len(collisions) == 2 and not is_found_collision:\n",
+        "                is_found_collision = True\n",
+        "                collision_index = t\n",
+        "\n",
+        "        print(f\"Iteration number {_}\")\n",
+        "        clear_output(wait=True)\n",
+        "\n",
+        "    if is_found_collision: break\n",
```

Comment:
It would be better not to put them in the same line.

Reply:
Fixed.